internal/impl/text/chunker/options.go

```diff
@@ -0,0 +1,181 @@
+// The MIT License
```

Comment:
apache2?

Reply:
ah this is an import nvm.

Reply:
Yeah it's a fork from langchain go.

Using that library as a dependency would be an option but it's got a massive number of transitive dependencies and this package is all we need right now (and it has very little in terms of deps)

Reply:
I will add additional comments to make this more clear